src/wp-content/themes/twentytwentythree/styles/block-out.json

```diff
@@ -39,6 +29,16 @@
 					"color": "#ff7e7e",
 					"name": "Tertiary",
 					"slug": "tertiary"
+				},
+				{
+					"color": "#ff5252",
+					"name": "Block out Red",
+					"slug": "block-out-red"
```

Comment:
I struggled to name some of these colors! I don't think the name in particular matters too much, but it would be good if the name described the color, or perhaps matched the personality of the style variation.